docker/pypi/wmagent-couchdb/couchdb-docker-run.sh

```diff
@@ -69,6 +69,11 @@ dockerOpts="
 --detach \
 --network=host \
 --rm \
+--user $UID:$GID \
+--volume='/etc/group:/etc/group:ro' \
+--volume='/etc/passwd:/etc/passwd:ro' \
+--volume='/etc/shadow:/etc/shadow:ro' \
+--volume='/etc/sudoers:/etc/sudoers:ro' \
```

Comment:
maybe we should also bind mount `/etc/sudoers.d`   because especially in the CERN machines, there are the actual lists of `sudo`  users (puppet managed of course)

Reply:
ok, that's fine, added